code/modules/mob/living/carbon/xenomorph/abilities/shrieker/shrieker_powers.dm

```diff
@@ -0,0 +1,219 @@
+/datum/action/xeno_action/activable/shriek/use_ability(atom/A)
+	var/mob/living/carbon/Xenomorph/xeno = owner
+	if (!istype(xeno))
+		return
+
+	if (!action_cooldown_check())
+		return
+	if (!check_plasma_owner())
+		return
+
+	if(!xeno.check_state())
+		return
+
+	if (xeno.mutation_type == SHRIEKER_NORMAL)
+		var/datum/behavior_delegate/shrieker/BD = xeno.behavior_delegate
+		if (!istype(BD))
+			return
+		if (!BD.use_internal_acid_ability(shriek_cost))
+			return
+
+	if (curr_effect_type == SHRIEKER_SHRIEK_BUFF)
+
+
+		playsound(xeno.loc, shriek_sound_effectt, 55, 0, status = 0)
+		xeno.visible_message(SPAN_XENOHIGHDANGER("[xeno] emits a piercing shriek!"))
+		xeno.create_shriekwave(color = "#07f707")
+		var/shriek_duration = 150
+		var/image/buff_overlay = get_busy_icon(ACTION_GREEN_POWER_UP)
+		var/mob/living/carbon/Xenomorph/Praetorian/P = owner
+		if (!(P.shriek_status_flags & SHRIEKER_SHRIEK_ENHANCED))
+			P.armor_modifier += XENO_ARMOR_MOD_MED
+			P.damage_modifier += XENO_DAMAGE_MOD_SMALL
+			P.recalculate_armor()
+			P.recalculate_damage()
+			P.shriek_status_flags |= SHRIEKER_SHRIEK_ENHANCED
+			to_chat(src, SPAN_XENOWARNING("Your shriek empowers you to strike harder!"))
+			buff_overlay.flick_overlay(P, 150)
+
+			spawn (shriek_duration)
+				P.armor_modifier -= XENO_ARMOR_MOD_MED
+				P.damage_modifier -= XENO_DAMAGE_MOD_SMALL
+				P.recalculate_armor()
+				P.recalculate_damage()
+				P.shriek_status_flags &= ~SHRIEKER_SHRIEK_ENHANCED
+				to_chat(src, SPAN_XENOWARNING("You feel the power of your shriek wane."))
+
+		else
+			to_chat(src, SPAN_XENOWARNING("Your shriek effects do NOT stack with other shrieks!"))
+
+		for(var/mob/living/carbon/Xenomorph/XX in view(6, xeno))
+			var/image/bufff_overlay = get_busy_icon(ACTION_GREEN_POWER_UP)
+			if (!(XX.shriek_status_flags & SHRIEKER_SHRIEK_ENHANCED))
+				XX.armor_modifier += XENO_ARMOR_MOD_MED
+				XX.damage_modifier += XENO_DAMAGE_MOD_SMALL
+				XX.shriek_status_flags |= SHRIEKER_SHRIEK_ENHANCED
+				XX.recalculate_armor()
+				XX.recalculate_damage()
+				bufff_overlay.flick_overlay(XX, 150)
+				to_chat(XX, SPAN_XENOWARNING("You feel empowered after heearing the shriek of [src]!"))
+
+				spawn (shriek_duration)
+					XX.armor_modifier -= XENO_ARMOR_MOD_MED
+					XX.damage_modifier -= XENO_DAMAGE_MOD_SMALL
+					XX.shriek_status_flags &= ~SHRIEKER_SHRIEK_ENHANCED
+					XX.recalculate_armor()
+					XX.recalculate_damage()
+					to_chat(XX, SPAN_XENOWARNING("You feel the effects of [src] wane!"))
+			else
+				to_chat(XX, SPAN_XENOWARNING("You can only be empowered by one shriek at once!"))
+
+	else if (curr_effect_type == SHRIEKER_SHRIEK_DEBUFF)
+		playsound(xeno.loc, shriek_sound_effectt, 55, 0, status = 0)
+		xeno.visible_message(SPAN_XENOHIGHDANGER("[xeno] emits a guttural shriek!"))
+		xeno.create_shriekwave(color = "#FF0000")
+		var/slow_duration = 40
```

Comment:
move this to the ability config variables whats the point of it here

also 4 SECONDS not 40